config/dev.webpack.config.js

```diff
@@ -10,7 +10,7 @@ const insightsProxy = {
 const webpackProxy = {
   deployment: process.env.BETA ? 'beta/apps' : 'apps',
   useProxy: true,
-  env: `stage-${process.env.BETA ? 'beta' : 'stable'}`, // for accessing prod-beta change this to 'prod-beta'
+  env: `ci-${process.env.BETA ? 'beta' : 'stable'}`, // for accessing prod-beta change this to 'prod-beta'
```

Comment:
Ah, you're doing this locally, this shouldn't be committed... I put up https://github.com/RedHatInsights/hac-core-frontend/pull/6 for that use-case